### ISSUE TITLE
[distributed] Invalid port added to Peer list

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio#3047]
```
$ minio server ~/test

Endpoint:  http://10.0.0.13:9000  http://127.0.0.1:9000  http://172.18.0.1:9000  http://172.17.0.1:9000
AccessKey: USWUXHGYZQYFYFFIT3RE 
SecretKey: MOJRH0mkL1IPauahWITSVvyDrQbEEIwljvmxdq03 
Region:    us-east-1
SQS ARNs:  arn:minio:sqs:us-east-1:2:amqp arn:minio:sqs:us-east-1:1:redis

Browser Access:
   http://10.0.0.13:9000  http://127.0.0.1:9000  http://172.18.0.1:9000  http://172.17.0.1:9000

Command-line Access: https://docs.minio.io/docs/minio-client-quickstart-guide
   $ mc config host add myminio http://10.0.0.13:9000 USWUXHGYZQYFYFFIT3RE MOJRH0mkL1IPauahWITSVvyDrQbEEIwljvmxdq03

Object API (Amazon S3 compatible):
   Go:         https://docs.minio.io/docs/golang-client-quickstart-guide
   Java:       https://docs.minio.io/docs/java-client-quickstart-guide
   Python:     https://docs.minio.io/docs/python-client-quickstart-guide
   JavaScript: https://docs.minio.io/docs/javascript-client-quickstart-guide

Drive Capacity: 126 GiB Free, 165 GiB Total
[:0 :9000]
ERRO[0015] Error sending peer update bucket policy to :0 - dial tcp :0: getsockopt: connection refused  cause=dial tcp :0: getsockopt: connection refused location=S3PeersUpdateBucketPolicy [s3-peer-client.go:214]
```

**Comment by harshavardhana** [comment]
Clearly we can see port is ":0"

**Comment by harshavardhana** [comment]
Fixed by #3048
